src/OrbitBase/include/OrbitBase/Profiling.h

```diff
@@ -6,17 +6,16 @@
 #define ORBIT_BASE_PROFILING_H_
 
 #ifdef _WIN32
-#include <Windows.h>
+#include <chrono>
 #else
 #include <stdint.h>
 #include <time.h>
 #endif
 
 #ifdef _WIN32
 [[nodiscard]] inline uint64_t MonotonicTimestampNs() {
-  __int64 time;
-  GetSystemTimeAsFileTime((FILETIME*)&time);
-  return static_cast<uint64_t>(time) * 100;
+  auto time_since_epoch = std::chrono::high_resolution_clock::now().time_since_epoch();
+  return std::chrono::duration_cast<std::chrono::nanoseconds>(time_since_epoch).count();
 }
 #else
 [[nodiscard]] inline uint64_t MonotonicTimestampNs() {
```

Comment:
iirc we want to keep this as clock_gettime(CLOCK_MONOTONIC) for compatibility with perf_event_open timestamps? Maybe add a comment here so that it is not merged with method above? Or even rename this method so that it is clearer why this is not implemented via <chrono> calls.